ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.13-nightly.yaml

```diff
@@ -212,6 +212,29 @@ tests:
     test:
     - ref: openshift-qe-ingress-perf
     workflow: rosa-aws-sts
+- as: perfscale-rosa-hcp-3zones-ingress-perf-3nodes-periodic
+  cron: 0 7 * * 2
+  steps:
+    cluster_profile: aws-perfscale-qe
+    env:
+      CHANNEL_GROUP: nightly
+      COMPUTE_MACHINE_TYPE: m5.2xlarge
+      DISABLE_WORKLOAD_MONITORING: "false"
+      ENABLE_AUTOSCALING: "false"
+      HOSTED_CP: "true"
+      INFRA_MACHINE_TYPE: c5.4xlarge
+      INFRA_NAME: infra
+      INFRA_NODE_LABELS: node-role.kubernetes.io/infra=
+      INFRA_NODE_TAINTS: node-role.kubernetes.io/infra=:NoSchedule
```

Comment:
Same here. Do we need to quote the value? Meaning, specify as "node-role.kubernetes.io/infra=:NoSchedule"?